packages/rocketchat-ui-master/client/main.js

```diff
@@ -184,7 +184,7 @@ Template.main.events({
 	},
 	'touchmove'(e, t) {
 		if (t.touchstartX != null) {
-			const [touch] = e.originalEvent.touches;
+			const touch = e.originalEvent.touches[0];
```

Comment:
This is needed because it seems that the TouchList object is not iterable on (some?) mobile platforms, thus raising an exception on every scroll event in the cordova app.

More details are here: #7075